app/src/main/java/com/android/sample/model/recipe/RecipeBuilder.kt

```diff
@@ -24,36 +24,36 @@ class RecipeBuilder {
    *
    * @param idMeal The ID of the recipe.
    */
-  fun setId(idMeal: String) = apply { this.idMeal = idMeal }
+  fun setId(idMeal: String) = apply { this.uid = idMeal }
 
   /**
    * Sets the name of the recipe.
    *
    * @param strMeal The name of the recipe.
    */
-  fun setName(strMeal: String) = apply { this.strMeal = strMeal }
+  fun setName(strMeal: String) = apply { this.name = strMeal }
 
   /**
    * Sets the category of the recipe.
    *
    * @param strCategory The category of the recipe.
    */
-  fun setCategory(strCategory: String) = apply { this.strCategory = strCategory }
+  fun setCategory(strCategory: String) = apply { this.category = strCategory }
```

Comment:
I think the parameter names should also be refactored to just `category` instead of `strCategory` to increase readability.

Reply:
Done!